class-parsely-recommended-widget.php

```diff
@@ -92,6 +92,7 @@ public function widget( $args, $instance ) {
 		$title = apply_filters( 'widget_title', $instance['title'] );
 
 		wp_enqueue_style( 'wp-parsely-style' );
+		wp_enqueue_script( 'jquery' );
```

Comment:
> Instead of enqueueing jQuery directly, can you add it as a dependency arg for the registration/enqueue of the now-external .js files?

The Widget script is not yet extracted from the PHP, so it has to be explicitly enqueued here.

Once we do pull it out, we'll definitely want to add it as a dependency to script (unless we remove the dependency entirely, of course).